service/src/test/resources/templates/ehr_extract_template.mustache

```diff
@@ -0,0 +1,57 @@
+<EhrExtract classCode="EXTRACT" moodCode="EVN">
+    <id root="{{ehrExtractId}}" />
+    <statusCode code="COMPLETE" />
+    <availabilityTime value="{{availabilityTime}}" />
```

Comment:
This is currently not mapped. According to the MiM we can populate it with the current timestamp. https://data.developer.nhs.uk/dms/mim/4.2.00/Domains/GP2GP/Tabular%20View/RCMR_HD030100UK04-NoEdit.htm

Correct format also available from that link